engine/src/main/java/com/redhat/cloud/notifications/db/repositories/DrawerNotificationRepository.java

```diff
@@ -17,18 +15,9 @@ public class DrawerNotificationRepository {
     EntityManager entityManager;
 
     @Transactional
-    public List<DrawerNotification> create(Event event, String users) {
-        return entityManager.createNativeQuery("SELECT * FROM insert_drawer_notifications(:users, :orgId, :eventId, :created)", DrawerNotification.class)
-            .setParameter("users", users)
-            .setParameter("orgId", event.getOrgId())
-            .setParameter("eventId", event.getId())
-            .setParameter("created", Timestamp.valueOf(event.getCreated())).getResultList();
-    }
-
-    @Transactional
-    public void createWithId(Event event, String drawerNotificationIdList) {
-        entityManager.createNativeQuery("CALL insert_drawer_notifications_with_id(:drawerNotificationIdList, :orgId, :eventId, :created)")
-            .setParameter("drawerNotificationIdList", drawerNotificationIdList)
+    public void create(Event event, String userIds) {
+        entityManager.createNativeQuery("CALL insert_drawer_notifications(:userIds, :orgId, :eventId, :created)")
```

Comment:
This calls `insert_drawer_notifications` while the SQL script contains `insert_drawer_notifications_with_id`. Is that normal?